src/player_comptask.c

```diff
@@ -2152,9 +2151,51 @@ long count_creatures_at_call_to_arms(struct Computer2 *comp)
     return count_player_list_creatures_of_model_matching_bool_filter(dungeon->owner, -1, creature_is_called_to_arms);
 }
 
-struct Thing *find_creature_for_call_to_arms(struct Computer2 *comp, long a2)
+static struct Thing *find_creature_for_call_to_arms(struct Computer2 *comp, TbBool prefer_high_scoring)
 {
-    return _DK_find_creature_for_call_to_arms(comp, a2);
+    struct Thing *thing;
+    int highest_score;
+    char state;
+    thing = INVALID_THING;
+    highest_score = INT_MAX;
+
+    for (struct Thing *i = thing_get(comp->dungeon->creatr_list_start); 
+        !thing_is_invalid(i); 
+        i = thing_get(creature_control_get_from_thing(i)->players_next_creature_idx))
+    {
+        struct CreatureControl *cctrl = creature_control_get_from_thing(i);
+
+        if ( (i->alloc_flags & TAlF_IsInLimbo) != 0 )
+            continue;
+        if ( (i->state_flags & TAlF_IsInMapWho) != 0 )
+            continue;
+        if ( i->active_state == CrSt_CreatureUnconscious )
+            continue;
+
+        if ( i->active_state == CrSt_MoveToPosition )
+            state = i->continue_state;
+        else
+            state = i->active_state;
+        struct StateInfo *stati = get_thing_state_info_num(state);
+
+        if ( (cctrl->spell_flags & CSAfF_Rebound) != 0 )
```

Comment:
I'm not seeing the original code in my file, but is this correct? Seems real strange that creatures with active rebound are excluded. CSAfF_CalledToArms for example would make more sense to me,...

Reply:
I see, yes it was split in 2 bytes and didn't take that into account should indeed be CSAfF_CalledToArms